### PR TITLE
INDY-1475: Add CSV generation to get_metrics

### PR DESCRIPTION
**Pull request by skhoroshavin**

### CODE REVIEW

File: scripts/get_metrics [hunk 1]
```diff
@@ -72,56 +80,107 @@ def make_copy_of_data(data_dir):
...
+        for ts, frame in sorted(stats.frames(), key=lambda v: v[0]):
+            row = [
+                ts.replace(microsecond=0),
+                frame.get(MetricsType.LOOPER_RUN_TIME_SPENT).avg,
+                frame.get(MetricsType.THREE_PC_BATCH_SIZE).avg,
+                frame.get(MetricsType.INCOMING_NODE_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.OUTGOING_NODE_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.INCOMING_CLIENT_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.OUTGOING_CLIENT_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.NODE_STACK_MESSAGES_PROCESSED).sum / stats.timestep.total_seconds(),
```
Comment (ashcherbakov) [inline scripts/get_metrics]: why do divide to time here? Do we consider all metrics per second?

**Comment by skhoroshavin** [comment]
Yes, some of them are per second, it gives more consistent results when
changing time windows.

вт, 31 июл. 2018 г., 10:44 ashcherbakov <notifications@github.com>:

File: scripts/get_metrics (hunk 1 above)
Comment (skhoroshavin) [reply scripts/get_metrics]: Yes, some of them are per second, it gives more consistent results when changing time windows.

**Comment by skhoroshavin** [comment]
Please, don't merge this until plenum is built and version in `config.py` is bumped

File: scripts/get_metrics [hunk 2]
```diff
@@ -72,56 +80,107 @@ def make_copy_of_data(data_dir):
...
+            "avg_three_pc_batch_size",
+            "avg_incoming_node_message_size",
+            "avg_outgoing_node_message_size",
+            "avg_incoming_client_message_size",
+            "avg_outgoing_client_message_size",
+            "node_stack_messages_processed",
+            "client_stack_messages_processed",
+            "three_pc_batch_count",
+            "requests_count",
+            "incoming_node_messages",
```
Comment (ashcherbakov) [inline scripts/get_metrics]: Please mention that this is per second

**Pull request merged**